Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM zonemaster/engine:local as build
+
+ARG version
```

Comment:
In regards to layer caching, wouldn't it make slightly more sense to put that argument after the _apt add_ and the _cpanm_?

Reply:
I thought the ARG didn't affect the caching until it was used. I'll take a closer look.